tests/plugin/docker-compose.base.yml

```diff
@@ -35,10 +35,9 @@ services:
   agent:
     image: apache/skywalking-python-agent:latest-plugin
     environment:
-      SW_AGENT_COLLECTOR_BACKEND_SERVICES: collector:19876
```

Comment:
Some of the env var names (like this one) were deliberately given that names to keep it consistent with other agent's configuration, although I agree you rename to a better one.

Reply:
oO I didn't see this thread until now, do you think we should tweak some naming to sync with java agent?

Reply:
> oO I didn't see this thread until now, do you think we should tweak some naming to sync with java agent?

From the point of python keyed parameter feature, it maybe inconvinient to have a long name, so I generally shortened them a bit. Anyway I will think about it and update some if needed. Next version I suppose we will provide an actual config.toml file instead of fully relying on env vars or code changes.

Reply:
> > oO I didn't see this thread until now, do you think we should tweak some naming to sync with java agent?
> 
> 
> 
> From the point of python keyed parameter feature, it maybe inconvinient to have a long name, so I generally shortened them a bit. 

Can we shorten the parameter name in code while keep the original env var name? I think the parameter name doesn't have to be the same as env var name right?

> Anyway I will think about it and update some if needed. Next version I suppose we will provide an actual config.toml file instead of fully relying on env vars or code changes.

We can have a toml file for configuration for sure. But all the configurations in that file should be overridable via env var, that would be convenient in Docker/Kubernetes use case.

Reply:
> 

Yeah it doens't have to be the same just I think it's a convention to have that consistent env_prefix_var as I seen in other python libraries. I will make some changes tmr to make important envvars back in sync with Java agent, meanwhile I think it's okay to sacrafice the length consideration in exchange for consistency (to be the same as envvar).